src/web/app/components/question-edit-form/question-edit-form.component.ts

```diff
@@ -371,6 +371,18 @@ export class QuestionEditFormComponent implements OnInit {
         modalRef.result.then(() => {
           this.saveExistingQuestionEvent.emit();
         }, () => {});
+      // } else if (this.model.questionDetails) {
```

Comment:
What is this for?

Reply:
It was supposed to be a separate modal message to warn user of the consequence of toggling... Will remove entirely.